hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1252,6 +1253,43 @@ public function upgrade_1036() {
     return TRUE;
   }
 
+  /**
+   * Create new option group and values
+   */
+  public function upgrade_1037() {
+    $groupValues = [
+      'As per contract',
+      'Change in contractual hours',
+      'Change in contract type'
+    ];
+
+    $result = civicrm_api3('OptionGroup', 'get', [
+      'name' => 'hrleaveandabsences_work_pattern_change_reason'
+    ]);
+
+    if ($result['count'] == 0) {
+      $optionParams = [
+        'name' => "hrleaveandabsences_work_pattern_change_reason",
+        'title' => "Leave and Absence Work Pattern Change Reason",
+        'is_active' => 1
+      ];
+
+      civicrm_api3('OptionGroup', 'create', $optionParams);
+
+      foreach ($groupValues as $key => $value) {
+        $opValueParams = [
+          'option_group_id' => 'hrleaveandabsences_work_pattern_change_reason',
```

Comment:
maybe it would be cleaner, and shorter to make 'hrleaveandabsences_work_pattern_change_reason' a variable, since it's long and is used twice